### PR TITLE
Adding NUCLEO_L552ZE_Q and DISCO_L562QE

### PR DESCRIPTION
**Pull request by jeromecoutant**
### Description

Hi

I would like to add 2 future ST boards based on STM32L5:
https://www.st.com/ko/microcontrollers/stm32l5x2.html?querycriteria=productId=LN2131

- NUCLEO_L552ZE_Q
- DISCO_L562QE

Thx

@lmestm @mikisch81 @orenc17 

### Pull request type

    [ ] Fix
    [ ] Refactor
    [x] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

### CODE REVIEW

**Comment by jeromecoutant** [comment]
@bridadan

File: src/mbed_os_tools/detect/platform_database.py [hunk 1]
```diff
@@ -170,6 +170,8 @@
         u"0835": u"NUCLEO_F207ZG",
         u"0839": u"NUCLEO_WB55RG",
         u"0840": u"B96B_F446VE",
+        u"0854": u"NUCLEO_L552ZE_Q",
+        u"0855": u"DISCO_L552QE",
```
Comment (bridadan) [inline src/mbed_os_tools/detect/platform_database.py]: This is currently entered into the online database as `DISCO-L552E`. Should the `Q` be in there?

Comment (screamerbg) [reply src/mbed_os_tools/detect/platform_database.py]: @jeromecoutant I'm okay to amend the database. Let me know of your preference for target name.

Comment (bridadan) [reply src/mbed_os_tools/detect/platform_database.py]: Could either of you let me know when the database has been changed? Once that's done we can bring this in, thanks.

**Comment by jeromecoutant** [comment]
Hi
PR title have been updated.
Thx

@svastm

**Comment by bridadan** [comment]
Thanks @jeromecoutant, just waiting on an update to the DB to match the entries here (@screamerbg @ashok-rao)

**Comment by bridadan** [comment]
Just a status update, waiting on `DISCO-L552E` to be renamed to `DISCO-L562QE` in the online DB before merging this.

**Comment by ashok-rao** [comment]
Apologies, I've been off for the last week too. Thanks @bridadan , DB now updated!

**Pull request merged**